addons/skin.estuary/xml/Includes_DialogVideoVersion.xml

```diff
@@ -0,0 +1,199 @@
+				<onleft condition="ControlGroup(200).HasFocus(202) + String.IsEmpty(Container(51).ListItem.Label)">200</onleft>
+				<itemgap>dialogbuttons_itemgap</itemgap>
+				<align>top</align>
+				<scrolltime tween="quadratic">200</scrolltime>
+				<include content="DefaultDialogButton">
+					<param name="id" value="21" />
+					<param name="label" value="$LOCALIZE[208]" />
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="id" value="22" />
+					<param name="label" value="$LOCALIZE[31612]" />
+					<param name="visible">ControlGroup(200).HasFocus(201)</param>
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="id" value="23" />
+					<param name="label" value="$LOCALIZE[31613]" />
+					<param name="visible">ControlGroup(200).HasFocus(202)</param>
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="id" value="24" />
+					<param name="label" value="$LOCALIZE[118]" />
+					
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="id" value="25" />
+					<param name="label" value="$LOCALIZE[31614]" />
+					<param name="visible">ControlGroup(200).HasFocus(201)</param>
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="id" value="26" />
+					<param name="label" value="$LOCALIZE[15015]" />
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="id" value="27" />
+					<param name="label" value="$LOCALIZE[13511]" />
+					<param name="visible">ControlGroup(200).HasFocus(201)</param>
+				</include>
+			</control>
+		</control>
+	</include>
+	<include name="DialogVideoVersionPlayLayout">
+		<control type="group">
+			<centertop>50%</centertop>
+			<centerleft>50%</centerleft>
+			<height>670</height>
+			<width>1020</width>
+			<include>Animation_DialogPopupVisible</include>
+			<include content="DialogBackgroundCommons">
+				<param name="width" value="1020" />
+				<param name="height" value="700" />
+				<param name="header_id" value="2" />
+			</include>
+			<include>DialogVideoVersionCommonLayout</include>
+		</control>
+	</include>
+	<include name="DialogVideoVersionCommonLayout">
+		<control type="grouplist" id="200">
+			<left>20</left>
+			<top>90</top>
+			<height>180</height>
+			<orientation>vertical</orientation>
+			<itemgap>dialogbuttons_itemgap</itemgap>
+			<onright condition="ControlGroup(200).HasFocus(201)">50</onright>
+			<onright condition="ControlGroup(200).HasFocus(202)">51</onright>
+			<include content="DefaultDialogButton">
+				<param name="id" value="201" />
+				<param name="visible">Window.IsActive(videoversion)| Window.IsActive(videoversionplay) + !String.IsEmpty(Container(50).ListItem.Label)</param>
+				<param name="enable">!String.IsEmpty(Container(51).ListItem.Label) | Window.IsActive(videoversion)</param>
+				<param name="label" value="Versions" />
```

Comment:
To be corrected - temporary hack still has not been removed, I need to set correct localised label of $LOCALIZE[40000]